contracts/interfaces/IDepositHandler.sol

```diff
@@ -3,4 +3,5 @@ pragma solidity 0.6.4;
 interface IDepositHandler {
     function deposit(uint8 destinationChainID, uint256 depositNonce, address depositer, bytes calldata data) external;
     function executeDeposit(bytes calldata data) external;
+    function setResourceIDAndContractAddress(bytes32, address) external;
```

Comment:
`GenericHandler.sol` has a `setResource` method that take arguments: 
```solidity
function setResource(
        bytes32 resourceID,
        address contractAddress,
        bytes4 depositFunctionSig,
        bytes4 executeFunctionSig
)
```

I think we could change this method in the interface to `function setResource(bytes resourceData) external;` and the handler should unpack their expected data